projects/observability/src/shared/dashboard/widgets/donut/donut-widget.model.ts

```diff
@@ -82,4 +98,13 @@ export class DonutWidgetModel {
       }))
     );
   }
+
+  private buildSeriesWithColors(series: DonutSeries[]): DonutSeries[] {
+    const colors = this.colorService.getColorPalette(DonutWidgetModel.DONUT_COLORS).forNColors(series.length);
```

Comment:
`this.colorService.getColorPalette(this.colorPalette)` - that assumes we change the default for this to be undefined rather than an empty array, which is an invalid palette.